docs/api.json

```diff
@@ -0,0 +1,1070 @@
+{
+  "title": "twine-api",
+  "version": "0.1",
+  "types": {
+    "organisationDetails": {
+      "id": "Number",
+      "name": "String",
+      "logoUrl": "String",
+      "address1": "String",
+      "address2": "String",
+      "townCity": "String",
+      "postCode": "String",
+      "coordinates": "String",
+      "region": "String",
+      "sector": "String",
+      "registeredAt": "String",
+      "lastActiveAt": "String",
+      "modifiedAt": "String",
+      "turnoverBand": "String",
+      "360GivingId": "String",
+      "fundingProgramme": "String",
+      "fundingBody": "String",
+      "subscription": "String"
+    },
+    "organisationTrainingSession": {
+      "trainingSessionAttendees": ["Number"],
+      "trainingSessionNotes": "String",
+      "eventDate": "String"
+    },
+    "adminUser": {
+      "id": "Number",
+      "userType": "String",
+      "name": "String",
+      "gender": "String",
+      "birthYear": "String",
+      "email": "String",
+      "phoneNumber": "String",
+      "postCode": "String",
+      "emailConsent": "Boolean",
+      "smsConsent": "Boolean",
+      "registeredAt": "String",
+      "lastActiveAt": "String",
+      "modifiedAt": "String",
+      "organisation": {
+        "organisationId": "Number",
```

Comment:
It felt more appropriate to add a reference to the org admins belong to rather than their full details. I don't think full org details need to be send as part of the user endpoint. This could either be removed or added to the other user types.

Reply:
OK, then we can just have `organisationId`. If we need the name we can make an additional request.